### DISCUSSION TITLE
members/3-ordinaryMembers/%E7%BB%8F%E6%B5%8E%E6%98%AF%E6%98%AF%E6%98%AF

### DISCUSSION THREAD

**Discussion by giscus** [seewo-geek/seewoGeekWikiComment#7] [General]
# members/3-ordinaryMembers/%E7%BB%8F%E6%B5%8E%E6%98%AF%E6%98%AF%E6%98%AF

希沃售后冲击部官方Wiki

https://wiki.misaka.space/members/3-ordinaryMembers/%E7%BB%8F%E6%B5%8E%E6%98%AF%E6%98%AF%E6%98%AF.html

**Comment by jjssszzr** [comment]
啊，真有两千字呀
